### DISCUSSION TITLE
Dashboard Start Problem

### DISCUSSION THREAD

**Discussion by RahatUllahDevopsEngineer** [timoschwarzer/gitlab-monitor#211] [General]
#

**Comment by timoschwarzer** [comment]
Unfortunately, I don't get what problem you're having. Could you explain the steps you're executing a bit more in detail?

**Comment by RahatUllahDevopsEngineer** [comment]
i  explain my problem in screenshot

**Comment by timoschwarzer** [comment]
What screenshot are you referring to?

**Comment by RahatUllahDevopsEngineer** [comment]
Waiting for your response

**Reply by timoschwarzer** [reply]
I asked you what screenshot you were referring to.

**Comment by RahatUllahDevopsEngineer** [comment]


**Reply by timoschwarzer** [reply]
Sorry, I do not know what you mean. Please describe, step by step, what you are trying to do.
